UndertaleModLib/Util/TextureWorker.cs

```diff
@@ -122,13 +122,14 @@ public static MagickImage ReadBGRAImageFromFile(string filePath)
         }
 
         /// <summary>
-        /// Performs a resize of the given image, if required, using bilinear interpolation. Always returns a new image.
+        /// Performs a resize of the given image, if required, using the specified interpolation (bilinear by default). Always returns a new image.
```

Comment:
Not sure. I believe the old method used wasn't nearest-neighbor, and there could be a benefit of it being very obvious you just tried to upscale a sprite on accident. In NN it can be slightly less obvious, especially if the aspect ratio only slightly differs. This would mostly affect the GUI though, which probably uses the default parameters currently.